packages/repository/src/types/index.ts

```diff
@@ -35,6 +37,8 @@ export {
   AnyType,
   ArrayType,
   UnionType,
+  ModelType,
+  ObjectType,
```

Comment:
IIUC, you have renamed `ObjectType<T>` to `DataObject<T>` in most places. Should  this code export `DataObject` instead or in addition to `ObjectType`?